schema/jsf-0.82.schema.json

```diff
@@ -15,7 +15,7 @@
             "signers": {
               "type": "array",
               "title": "Signature",
-              "description": "Unique top level property for Multiple Signatures. (multisignature)",
+              "description": "Unique top level property for Multiple Signatures. (multisignature). Can include both classical and post-quantum signatures.",
```

Comment:
Question: Ideally, this shouldn't be allowed, since it creates a weaker link. Perhaps this is needed to allow everyone to upgrade?